### PR TITLE
Feature simplify

### PR DESCRIPTION
**Pull request by paulfantom**
Added:

simpler vhost creation
more tests
custom error pages
prevention against BREACH attack (no gzip)
websocket autodetection and support out-of-the-box
custom vhost for /nginx_status location on port 8060 (can be customized with nginx_metrics_port variable)
enabled tests for nginx metrics exporter
tests moved to newer testinfra

### CODE REVIEW

File: templates/nginx-status.j2 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+server  {
```
Comment (tgrzesik) [inline templates/nginx-status.j2]: same content in nginx-status.conf.j2

Comment (tgrzesik) [inline templates/nginx-status.j2]: same content in nginx-status.conf.j2

**Review by tgrzesik — COMMENTED** [review]
requested changes

**Review by tgrzesik — CHANGES_REQUESTED** [review]
requested changes

**Comment by ghost** [comment]
@tgrzesik fixed

**Review by tgrzesik — APPROVED** [review]

**Pull request merged**